documentation/library_guide/extension_api.rst

```diff
@@ -255,17 +256,20 @@ These algorithms are declared in ``oneapi::dpl::experimental::ranges`` namespace
 In order to make these algorithm available the ``<oneapi/dpl/ranges>`` header should be included.
 Use of the range-based API requires C++17 and the C++ standard libraries coming with GCC 8.1 (or higher) or Clang 7 (or higher).
 
-The following viewable ranges are declared in ``oneapi::dpl::experimental::ranges`` namespace. Only those are allowed to use as ranges
+The following views are declared in ``oneapi::dpl::experimental::ranges`` namespace. Only those are allowed to use as ranges
```

Comment:
Please add "the": The following views are declared in **the** ``oneapi::dpl::experimental::ranges`` namespace.